graylog2-web-interface/src/views/components/messagelist/MessageFields/hooks/useSendFavoriteFieldTelemetry.ts

```diff
@@ -27,13 +27,15 @@ const useSendFavoriteFieldTelemetry = () => {
   const widget = useContext(WidgetContext);
   const sendTelemetry = useSendTelemetry();
 
+  const widgetType = widget?.type ?? 'permalink';
```

Comment:
@maxiadlovskii I'm not sure this is the correct approach to fix this, but noticed it when testing some investigations updates.